spec/integration/plos_details_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper.rb'
+
+describe PlosDetails, :vcr do
+  let(:plos_paper) do
+    create(:plos_paper, id: 1, reader: 0, doi: '10.1371/journal.pone.0089204')
+  end
+  let(:job) {PlosDetails.new}
+
+  it 'updates the view count' do
+    before = plos_paper.reader
+
+    VCR.use_cassette 'plos_detail' do
+      job.perform plos_paper.id
+    end
+    expect(plos_paper.reload.reader).not_to eq(before)
```

Comment:
Since you set `reader` to 0 when setting up the paper, you don't need to check against the before here. Checking that `reader` has a value after running the job is enough. Assinging `before` is also not necessary, then.